simulator/index.md

```diff
@@ -33,7 +33,7 @@ which will be available to your robot code during the competition. Note that for
 local development you will need to install these yourself.
 
 If there are other libraries you would like included, please let us know
-[via the forums](/forum) and include a link to the package on [PyPI](https://pypi.org/).
+[in the discord](https://discord.com/channels/775497131057741836/777582218662510612) and include a link to the package on [PyPI](https://pypi.org/).
```

Comment:
Maybe pull out a footer link to avoid duplication? Or is this deliberately a different channel? (If so, could we mention the channel name in the link text?)

Reply:
540db97